LICENSE

```diff
@@ -0,0 +1,171 @@
+Copyright (c) 2015-2017 Okta Inc.
```

Comment:
I usually do 2020-Present here so I don't have to change it again.

Reply:
updated